### DISCUSSION TITLE
InternalServerError

### DISCUSSION THREAD

**Discussion by makegeekca** [georged/code-connectors#3] [Q&A]
Hello, I'm using this in a flow, trying to recreate Amey Holden's Marketing List creation from csv file. https://www.ameyholden.com/articles/import-spreadsheet-csv-leads-contact-marketing-list-dynamics-365

I was able to install this via powershell, not via powerapps.com 

When I test with 2 lines in my csv (header and one line of data), I get InternalServerError and it continuously retries.

Any thoughts?
![image](https://github.com/georged/code-connectors/assets/5420853/faa92edb-a2eb-4115-b756-3de51438eeb7)

**Comment by georged** [comment]
@makegeekca can I have a sample CSV that generates the error?

**Comment by makegeekca** [comment]
[
[Newsletter-InitialList-DVTest.csv](https://github.com/georged/code-connectors/files/15403536/Newsletter-InitialList-DVTest.csv)
](url)

No problem at all. Here you go.

**Comment by georged** [comment]
@makegeekca it works in my environment:

![image](https://github.com/georged/code-connectors/assets/928254/26ddf78c-c076-4c94-bab7-e935709e104c)

The only thing I can suggest is to go into test tab in the custom connector and then use log statements in the code try to figure out what's wrong. I would also suggest removing retries on that step. Once it fails, it's unlikely to succeed on retries.
